getting_started/workflow/assets/importing_scenes.rst

```diff
@@ -172,6 +172,10 @@ most 3D DCCs don't have the same material options as those present in Godot.
 When materials are built-in, they will be lost each time the source scene
 is modified and re-imported.
 
+.. note::
+
+    Godot will not reimport materials that are stored in external files.
```

Comment:
```suggestion
    Godot will not reimport materials that are stored in external files
    unless you remove the associated `.material` file before reimporting.
    
    To force reimporting materials every time the 3D scene is reimported,
    change the material storage mode in the 3D scene by selecting it in
    the FileSystem dock, then go the Import dock and set
    **Material > Storage** to **Built-In** instead of **Files**.
```